src/main/java/listeners/VideoRecorder.java

```diff
@@ -0,0 +1,90 @@
+package listeners;
+
+import org.monte.media.Format;
+import org.monte.media.math.Rational;
+import org.monte.screenrecorder.ScreenRecorder;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+import webdriver.DriverManager;
+import ws.schild.jave.*;
+
+import java.awt.*;
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static org.monte.media.FormatKeys.*;
+import static org.monte.media.VideoFormatKeys.*;
+
+public class VideoRecorder {
+
+    private final String RECORD_DIRECTORY = "videos\\";
+    private ScreenRecorder screenRecorder;
+
+    public void startRecording() {
+        try {
+            GraphicsConfiguration gc = GraphicsEnvironment
+                    .getLocalGraphicsEnvironment().getDefaultScreenDevice()
+                    .getDefaultConfiguration();
+            File dir = new File(RECORD_DIRECTORY);
+            Point point = DriverManager.getDriver().manage().window().getPosition();
+            Dimension dimension = DriverManager.getDriver().manage().window().getSize();
+            Rectangle rectangle = new Rectangle(point.x, point.y,
+                    dimension.width, dimension.height);
+            this.screenRecorder = new ScreenRecorder(gc, rectangle,
+                    new Format(MediaTypeKey, MediaType.FILE, MimeTypeKey,
+                            MIME_AVI),
+                    new Format(MediaTypeKey, MediaType.VIDEO, EncodingKey,
+                            ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE,
+                            CompressorNameKey,
+                            ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE, DepthKey,
+                            24, FrameRateKey, Rational.valueOf(15), QualityKey,
+                            1.0f, KeyFrameIntervalKey, 15 * 60), new Format(
+                    MediaTypeKey, MediaType.VIDEO, EncodingKey,
+                    "black", FrameRateKey, Rational.valueOf(30)), null,
+                    dir);
+            this.screenRecorder.start();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public File stopRecording(String recordName) {
+        File newFileName = null;
+        try {
+            this.screenRecorder.stop();
+            if (recordName != null) {
+                SimpleDateFormat dateFormat = new SimpleDateFormat(
+                        "yyyy-MM-dd HH.mm.ss");
+                newFileName = new File(String.format("%s%s%s.mp4",
```

Comment:
why is variable of type 'File' called 'newFileName' ? newFileName should store file name which is string